v1.10.0 Build on TypeScript 2.9.2 Reporting Error TS1039

It's all right if there's not an intent to support TypeScript v2.9 on this release, but I did want to raise that the latest minor release caused a build failure against that language version, which I wouldn't have expected.

Builds are reporting:

> ERROR in node_modules/oidc-client/index.d.ts(41,26): error TS1039: Initializers are not allowed in ambient contexts.

--- a/index.d.ts
+++ b/index.d.ts
@@ -38,11 +38,11 @@ export class InMemoryWebStorage {
 }
 
 export class Log {
-  static readonly NONE = 0;
-  static readonly ERROR = 1;
-  static readonly WARN = 2;
-  static readonly INFO = 3;
-  static readonly DEBUG = 4;
+  static readonly NONE: 0;
+  static readonly ERROR: 1;
+  static readonly WARN: 2;
+  static readonly INFO: 3;
+  static readonly DEBUG: 4;
 
   static reset(): void;
 

